.npmrc

```diff
@@ -1 +1,3 @@
-link-workspace-packages=false
\ No newline at end of file
+auto-install-peers=false
+strict-peer-dependencies=true
+disallow-workspace-cycles=true
```

Comment:
normally i'd prefer to keep the defaults for Pnpm stuff.

Also, this stuff is always very sensitive to touch, because the wrong configuration often breaks deploy systems (Vercel, Render.com and changesets). Happened already 4-5 times since the start of the project.

Reply:
I think it'd make sense to revert auto-install-peers, but the other two should just make the "linting" stricter.

Reply:
`strict-peer-dependencies` ehh, i don't think this is much practical. We have mismatched peer-deps all the time, because we depend on a lot of deps (that themselves have other deps and so on). This would cause a lot of issues i think. Also 90% of mismatched peer-deps are usually harmless false-positives.

`disallow-workspace-cycles` i had no idea about this setting. Sounds like something i would want on every project, not sure why this is not enabled by default, but this is definitely a good one 👍

Reply:
Well, both npm and Yarn set `strict-peer-dependencies` by default, so setting it forces you to go through the experience of a non-pnpm user and it's easily worked around with [`pnpm.peerDependencyRules`](https://pnpm.io/package_json#pnpmpeerdependencyrules). Additionally, the reasoning for `auto-install-peers=false` was that Yarn sets it by default (correctly, in my opinion), so it was just dogfooding the Yarn experience.